sam-react-app/coreconcept/src/counter/Counter.tsx

```diff
@@ -0,0 +1,76 @@
+import React, {ChangeEvent, Component, createElement} from "react";
+
+type CounterComponentState = {
+    counter: number;
+}
+type CounterComponentProps = {
+    initialValue: string;
+}
+class Counter extends Component<any, CounterComponentState> {
```

Comment:
Since you type state you might as well type props. It will replace generic parameter `any`.

Reply:
Done